ims/tgt_target.temp

```diff
@@ -1,7 +1,7 @@
 <target ${target_name}>
     driver iscsi
     bs-type rbd
-    backing-store boot-disk-prototype/${target_name}
+    backing-store ${pool}/${target_name}
     bsopts ""conf=${ceph_config};id=${ceph_user}""
     write-cache off
     initiator-address ALL
```

Comment:
I had something like tgt could be backed by anything like ceph or gluster or someother filesystem so, somewhere backend should be specified is what I thought.